src/data/corpus_sqlite.cpp

```diff
@@ -114,7 +114,7 @@ SentenceTuple CorpusSQLite::next() {
     SentenceTuple tup(curId);
 
     for(size_t i = 0; i < files_.size(); ++i) {
-      auto line = select_->getColumn(i + 1);
+      auto line = select_->getColumn((int)(i + 1));
```

Comment:
`int i`

Reply:
I had the choice between this fix (only one cast) or the fix you propose which implies 3 casts below in lines 119, 121 and 124.